stubs/openpyxl/openpyxl/styles/alignment.pyi

```diff
@@ -14,7 +15,7 @@ vertical_aligments: tuple[_VerticalAlignmentsType, ...]
 
 class Alignment(Serialisable):
     tagname: str
-    __fields__: Incomplete
+    __fields__: ClassVar[tuple[str, ...]]
```

Comment:
I can't seem to find `__fields__` in the implementation, looking at git HEAD. Same for `Side` and `Border` below.

Reply:
Tag 3.1.2 https://foss.heptapod.net/openpyxl/openpyxl/-/blob/3.1.2/openpyxl/styles/alignment.py#L21

Seems like it will disappear in 3.2